homeassistant/components/isy994/cover.py

```diff
@@ -49,16 +64,28 @@ def is_closed(self) -> bool:
             return None
         return self._node.status == 0
 
+    @property
+    def supported_features(self):
+        """Flag supported features."""
+        return SUPPORT_OPEN | SUPPORT_CLOSE | SUPPORT_SET_POSITION
+
     def open_cover(self, **kwargs) -> None:
         """Send the open cover command to the ISY994 cover device."""
-        if not self._node.turn_on(val=100):
```

Comment:
Is this a breaking change since we no longer default to sending `100`?

Reply:
I'd say yes, but I can't find any device that actually uses UOM 97 (which was the only thing previously represented as a cover). I'll guard against it though to avoid the BC just in case.